Mathlib/Data/Set/Pointwise/Basic.lean

```diff
@@ -297,7 +297,7 @@ protected def mul : Mul (Set α) :=
 scoped[Pointwise] attribute [instance] Set.mul Set.add
 
 @[to_additive (attr := simp)]
-theorem image2_mul : image2 Mul.mul s t = s * t :=
+theorem image2_mul : image2 (· * ·) s t = s * t :=
```

Comment:
Why is this change needed?

Reply:
Otherwise, the proof of `star_mul` gets more complicated since at some point, one needs to prove that 
```(fun a b : α ↦ (Mul.mul a b)⋆) = (fun a b : α ↦ b⋆ * a⋆) ```
Note that I was the one that did the porting of `Data/Pointwise/Basic` so it is likely that I wrote the wrong statement. The original mathlib3 lemma is:
```lemma image2_mul : image2 has_mul.mul s t = s * t := rfl```

This change was suggested by @Ruben-VandeVelde